calendar-bundle/src/Resources/contao/classes/Events.php

```diff
@@ -283,13 +281,21 @@ protected function addEvent($objEvents, $intStart, $intEnd, $intBegin, $intLimit
 			}
 		}
 
-		// Tag the response
+		// Tag the event and its child content elements (see #2137). Calendars
+		// will be tagged separately in the related front end modules.
 		if (System::getContainer()->has('fos_http_cache.http.symfony_response_tagger'))
 		{
-			/** @var ResponseTagger $responseTagger */
-			$responseTagger = System::getContainer()->get('fos_http_cache.http.symfony_response_tagger');
-			$responseTagger->addTags(array('contao.db.tl_calendar_events.' . $objEvents->id));
-			$responseTagger->addTags(array('contao.db.tl_calendar.' . $objEvents->pid));
+			$arrTags = array('contao.db.tl_calendar_events.' . $objEvents->id);
+
+			if ($objElements = ContentModel::findPublishedByPidAndTable($objEvents->id, 'tl_calendar_events'))
```

Comment:
Why are you tagging the content elements here? They tag themselves in `Controller::getContentElement()`.

Reply:
Ah, nice. 😎 Adjusted in 4b61ac2659d1dbb4d2809edd52acb6c9d468c4b0.